### ISSUE TITLE
Remove the black box replacing thumbnails when "load thumbnails" is disabled

### ISSUE THREAD

**Issue by lucanomax** [TeamNewPipe/NewPipe#4641]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
When you disable _load thumbnails_ there is a black box replacing the thumbnail, I think removing that and using the free space to display title and other infos with bigger font It's better to have a more minimal and compact UI.

#### Is your feature request related to a problem? Please describe it
It's not a problem, but the UI would for sure benefit from this change.

#### How will you/everyone benefit from this feature?
If you don't need thumbnails, having a black box next to the video title it's weird and ugly. Having a compact view, without many objects on screen, It's for sure more clean and nice to view without a black hole on the left.

**Comment by andrei-belkin** [comment]
I agree. As of v0.21.8 it looks like ![this](https://user-images.githubusercontent.com/59848424/128641436-3c43dcc6-7f40-42cc-9eb0-3354e0ce9807.png) - a lot of wasted, unused space.
